src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -319,7 +322,12 @@ def get(self, **kwargs):
         result = self._intersection_of_request_info(request_info)
         if len(result) == 0:
             return []
-        return rows([result])
+        
+        if common_dict == 1:
+            response_list = result.values()
```

Comment:
It doesn't have identical structure by nature. i.e. TaskChain has different structure than MC. If you need exact same structure, we first need a survey what information they need which has common for all the request and send only those.  RequestName field is in the value with current api.